trace/trace_test.go

```diff
@@ -246,11 +245,11 @@ func TestNestedTraceLog(t *testing.T) {
 			},
 			sampleTrace: &Trace{
 				name: "Sample Trace",
-				nestedTrace: []*Trace{
-					{
+				stepsTraces: []stepTrace{
+					&Trace{
 						startTime:   time.Now(),
 						name:        "msg1",
-						nestedTrace: []*Trace{{name: "nested1", startTime: time.Now()}},
+						stepsTraces: []stepTrace{&Trace{name: "nested1", startTime: time.Now()}},
 					},
 				},
 			},
```

Comment:
We need nested trace cases in `TestLogIfLong` (or in a new test function that test `LogIfLong` for nested traces).

Reply:
Okay, I will add those